lib/patch_ruby/api/orders_api.rb

```diff
@@ -166,9 +165,10 @@ def create_order_with_http_info(create_order_request, opts = {})
     # @param [Hash] opts the optional parameters
     # @option opts [PlaceOrderRequest] :place_order_request
     # @return [OrderResponse]
-    def place_order(id, place_order_request = {}, opts = {})
-      _place_order_request = Patch::PlaceOrderRequest.new(place_order_request)
-      data, _status_code, _headers = place_order_with_http_info(id, opts.merge!({place_order_request: place_order_request}))
+    def place_order(order_id, opts = {})
+      place_order_request = Patch::PlaceOrderRequest.new(opts)
+      opts[:place_order_request] = place_order_request
+      data, _status_code, _headers = place_order_with_http_info(order_id, opts)
```

Comment:
@holtbp finding out after the fact, but it looks like it's a change you generated manually, is it?